From 3247b0b309fd9b14894c973a1987597e7ab23937 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:10 -0400
Subject: [redhat] kabi: Adding symbol netdev_rx_handler_register (net/core/dev.c)

Message-id: <070ed0754fad6e0f4266d1df8794f438b7436503.1410510773.git.poros@redhat.com>
Patchwork-id: 96029
O-Subject: [PATCH RHEL7.1 130/233] kabi: Adding symbol netdev_rx_handler_register (net/core/dev.c)
Bugzilla: 1077335
RH-Acked-by: Jiri Benc <jbenc@redhat.com>

Bugzilla: 1077335
    https://bugzilla.redhat.com/show_bug.cgi?id=1077335

    Description:
    ============
    Invoked for each port specified to be on N1KV.
    This registers our KLM rx handler for packets received on these ports.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/netdev_rx_handler_register
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/netdev_rx_handler_register
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/netdev_rx_handler_register

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/netdev_rx_handler_register  b/redhat/kabi/kabi-module/kabi_ppc64/netdev_rx_handler_register 
new file mode 100644
index 0000000..7b1620a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/netdev_rx_handler_register  
@@ -0,0 +1,2 @@
+#1-
+0xc96149f5 netdev_rx_handler_register vmlinux EXPORT_SYMBOL_GPL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/netdev_rx_handler_register  b/redhat/kabi/kabi-module/kabi_s390x/netdev_rx_handler_register 
new file mode 100644
index 0000000..6fdb411
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/netdev_rx_handler_register  
@@ -0,0 +1,2 @@
+#1-
+0x557464cc netdev_rx_handler_register vmlinux EXPORT_SYMBOL_GPL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/netdev_rx_handler_register  b/redhat/kabi/kabi-module/kabi_x86_64/netdev_rx_handler_register 
new file mode 100644
index 0000000..2edeab4
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/netdev_rx_handler_register  
@@ -0,0 +1,2 @@
+#1-
+0x31e9790f netdev_rx_handler_register vmlinux EXPORT_SYMBOL_GPL
-- 
1.7.1